packages/web-components/src/components/logo-grid/logo-grid.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { css, customElement, html, property } from 'lit-element';
+import settings from 'carbon-components/es/globals/js/settings';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings';
+import styles from './logo-grid.scss';
+import DDSContentBlock from '../content-block/content-block';
+import '../content-block/content-block-heading';
+import StableSelectorMixin from '../../globals/mixins/stable-selector';
+
+const { prefix } = settings;
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * Logo grid.
+ *
+ * @element dds-logo-grid
+ */
+@customElement(`${ddsPrefix}-logo-grid`)
+class DDSLogoGrid extends StableSelectorMixin(DDSContentBlock) {
+  static get stableSelector() {
+    return `${ddsPrefix}--logo-grid`;
+  }
+
+  static get hideBorderClass() {
+    return `${prefix}--logo-grid__no-border`;
+  }
+
+  @property({ attribute: 'hide-border', reflect: true, type: Boolean })
+  hideBorder = false;
+
+  protected updated() {
+    this.classList.toggle((this.constructor as typeof DDSLogoGrid).hideBorderClass, this.hideBorder);
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  protected _renderContent() {
+    return html`
+      <div class="bx--content-block__children">
+        <div class="bx--logo-grid__row">
+          <slot></slot>
+        </div>
+      </div>
+    `;
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  protected _renderCopy() {
+    return '';
+  }
+
+  // `styles` here is a `CSSResult` generated by custom WebPack loader
+  static get styles() {
+    return css`${super.styles}${styles}`;
+  }
+}
```

Comment:
Nitpick for aligning with other classes in the codebase: Could you please re-group properties/methods, in the following order?

1. Protected methods
2. Public properties
3. Static properties